ansible/roles/osdsdock/scenarios/ceph.yml

```diff
@@ -80,26 +108,37 @@
     - ansible all -m ping -i ceph.hosts
     - ansible-playbook site.yml -i ceph.hosts | tee /var/log/ceph_ansible.log
   args:
-    chdir: /opt/ceph-ansible
+    chdir: /opt/ceph-ansible 
+  when: res["results"][0]["rc"] == 1 or service_ceph_mon_status.rc == 1 or service_ceph_osd_status.rc == 1
+
+- name: Configure Ceph-OSD
+  shell: "{{ item }}" 
+  become: true
+  ignore_errors: true
+  with_items:
+    - ceph-disk prepare --bluestore /dev/sdb #Change this device to your device for eg. /dev/sdv, /dev/sdc etc
```

Comment:
You need to import the device name from the parameters defined in `group_vars/ceph/all.yml`, one example is `{{ devices }}`